website/docs/d/datastore_cluster.html.markdown

```diff
@@ -44,6 +44,8 @@ The following arguments are supported:
   This can be omitted if the search path used in `name` is an absolute path.
   For default datacenters, use the id attribute from an empty
   `vsphere_datacenter` data source.
+* `datastores` - (Optional) The names of the datastores included in the specific 
+  cluster.
```

Comment:
```suggestion
* `datastores` - The names of datastores included in the datastore cluster.
```

Reply:
You are right. I meant to add datastores to the exported attributes. I also reformed the id attribute explanation now that this is a list.